File: mobile/src/main/java/com/windscribe/mobile/email/AddEmailView.java
/*
 * Copyright (c) 2021 Windscribe Limited.
 */

package com.windscribe.mobile.email;

public interface AddEmailView {

    void gotoWindscribeActivity();

    void hideSoftKeyboard();

    void prepareUiForApiCallFinished();

    void prepareUiForApiCallStart();

    void setUpLayout(String title);

    void showInputError(String errorText);

    void showToast(String toastString);

}

File: mobile/src/main/java/com/windscribe/mobile/email/AddEmailView.kt
/*
 * Copyright (c) 2021 Windscribe Limited.
 */
package com.windscribe.mobile.email

interface AddEmailView {
    fun gotoWindscribeActivity()
    fun hideSoftKeyboard()
    fun prepareUiForApiCallFinished()
    fun prepareUiForApiCallStart()
    fun setUpLayout(title: String)
    fun showInputError(errorText: String)
    fun showToast(toastString: String)
}